.circleci/config.yml

```diff
@@ -38,10 +30,36 @@ commands:
       - run:
           name: Set the PATH env variable
           command: |
-            # Also put the Rust LLVM tools into the PATH.
-            echo 'export PATH="$HOME:~/.cargo/bin:~/.rustup/toolchains/<< pipeline.parameters.stable-version >>-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/bin:$PATH"' | tee --append $BASH_ENV
+            # Also put the Rust LLVM tools into the PATH. `rustc` hooks into the rust-toolchain at the root of the repo
+            echo "export PATH=$HOME:~/.cargo/bin:$(dirname $(find $(rustc --print sysroot) -name 'rust-lld')):$PATH" | tee -a $BASH_ENV
```

Comment:
It looks like this fails on the Linux target with the following error:
```
/bin/bash: line 1: rustc: command not found
dirname: missing operand
Try 'dirname --help' for more information.
```
Perhaps we could omit the `set_env_path` command from Linux jobs if `~/.cargo/bin` is already in the PATH. Currently it looks like it is run twice, once in `cargo_fetch` and once in `linux`/`linux_release`